### PR TITLE
use new generator to fix accept header

### PR DESCRIPTION
**Pull request by dlsteuer**
This fixes the issue where post/put requests could set the correct version in the accept header.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,13 @@
-# Dyspatch Ruby API Client
+# dyspatch_client
```
Comment (phil-ma) [inline README.md:1]:
```suggestion
# Dyspatch Ruby API Client
```

**Review by phil-ma — APPROVED** [review]
Thanks @dlsteuer!

File: README.md (hunk 1 above)
Comment (dlsteuer) [reply README.md:1]: The only issue with this, is this file is generated, so any fixes for this should be handled as part of the readme updates in another task.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,13 @@
 # Dyspatch Ruby API Client
```
Comment (phil-ma) [inline README.md]:
```suggestion
# dyspatch_client
```

File: README.md (hunk 1 above)
Comment (phil-ma) [reply README.md:1]: Sounds good! Reverted back

**Pull request merged**